dask/highlevelgraph.py

```diff
@@ -346,7 +346,7 @@ def __dask_distributed_pack__(
           - Serializable by msgpack (notice, msgpack converts lists to tuples)
           - All remote data must be unpacked (see unpack_remotedata())
           - All keys must be converted to strings now or when unpacking
-          - All tasks must be serialized (see dumps_task())
+          - All tasks must be typesetted (see typeset_dask_graph())
```

Comment:
```suggestion
          - All tasks must be typeset (see typeset_dask_graph())
```
I have no idea if "typeset" is correct here, but I have a feeling that "typesetted" is not a word.

Reply:
> I have a feeling that "typesetted" is not a word

Heheh, I think you are right! :)